ledger/metrics/src/test/lib/scala/com/daml/metrics/api/testing/TestingInMemoryMetrics.scala

```diff
@@ -0,0 +1,162 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```

Comment:
To conform with naming conventions, I'd suggest renaming this source file to match the name of the factory class.